_styles/style.css

```diff
@@ -223,15 +220,15 @@ main {
 	height: auto;
 	max-width: 400px;
 	border-radius: 25px;
-	margin: 0 auto;
+	margin: auto;
 }
 
 .project-info {
 	width: 90%;
-	max-width: 400px;
+	max-width: 70ch;
```

Comment:
using different unit, not sure if it matters if you use different units for relativity